src/content/App/App.types.ts

```diff
@@ -1,52 +1,51 @@
-import { defaultColorList } from "../../helpers/courseColors";
-
 export enum SectionType {
-    lecture,
-    tutorial,
-    lab
+  lecture,
+  tutorial,
+  lab,
 }
 
 export enum Term {
-    summerOne,
-    summerTwo,
-    summerFull,
-    winterOne,
-    winterTwo,
-    winterFull,
+  summerOne,
+  summerTwo,
+  summerFull,
+  winterOne,
+  winterTwo,
+  winterFull,
 }
 
 export enum Views {
-    calendar,
-    settings
+  calendar,
+  settings,
 }
 
+export const Term_String_Map: { [key in Term]: string } = {
+  [Term.summerOne]: "S1",
+  [Term.summerTwo]: "S2",
+  [Term.summerFull]: "S",
+  [Term.winterOne]: "W1",
+  [Term.winterTwo]: "W2",
+  [Term.winterFull]: "W",
+};
 
-export const Term_String_Map: { [key in Term]: string} = {
-    [Term.summerOne]: 'S1',
-    [Term.summerTwo]: 'S2',
-    [Term.summerFull]: 'S',
-    [Term.winterOne]: 'W1',
-    [Term.winterTwo]: 'W2',
-    [Term.winterFull]: 'W',
-}
+export type SupplementaryData = {
+  instructors: string[];
+  locations: string[];
+};
 
 export type SectionDetail = {
-    term: Term,
-    days: string[]
-    startTime: string,
-    endTime: string,
-    location?: string,
-    dateRange: string   
-}
+  term: Term;
+  days: string[];
+  startTime: string;
+  endTime: string;
+  dateRange: string;
```

Comment:
Do we need both daterange and term?

Reply:
yes. daterange is needed for stuff like external calendar. Should not be an issue to have both now as going from sync storage to local takes our quota from 100kb -> 10mb